src/routes/auth/user.ts

```diff
@@ -68,32 +67,36 @@ userAuthRouter.post("/login", async (req, res) => {
   if (!valid) return res.status(400).json(errors);
 
   // Authenticate user
-
   // checking if user exists in DB
   const userEmailDoc = await UserModel.findOne({ email: userLoginData.email }).exec();
   if (!userEmailDoc) return res.status(400).json({ login: "email or password is incorrect" });
   else if (!(await userEmailDoc.checkPassword(userLoginData.password))) {
     return res.status(400).json({ login: "email or password is incorrect" });
   }
+  try {
+    // Return auth token;
+    const accessToken = jwt.sign({ email: userEmailDoc.email, username: userEmailDoc.username }, process.env.JWT_SECRET_USER);
 
-  // Return auth token;
-  let accessToken;
-  const TOKEN = process.env.ACCESS_TOKEN_SECRET;
-  TOKEN ? (accessToken = jwt.sign({ email: userEmailDoc.email, username: userEmailDoc.username }, TOKEN)) : console.log("No secret access token in env");
-
-  res.status(200).json({ jwt: `Bearer ${accessToken}` });
+    res.status(200).json({ jwt: `Bearer ${accessToken}` });
+  } catch (err) {
+    console.log(err);
+  }
 });
 
 //user whomai route
 userAuthRouter.post("/me", authDataOnlyMiddleware, async (req, res) => {
-  const userDoc = await UserModel.findOne({ username: res.locals.user.username }).exec(); // look for the user in db
-
-  // return data
-  res.status(200).json({
-    email: userDoc?.email,
-    username: userDoc?.username,
-    private_profile: userDoc?.private_profile,
-    liked_projects: userDoc?.liked_projects,
-    banned: userDoc?.banned,
-  });
+  try {
+    const userDoc = await UserModel.findOne({ username: res.locals.user.username }); // look for the user in db
+
+    // return data
+    res.status(200).json({
+      email: userDoc?.email,
+      username: userDoc?.username,
+      private_profile: userDoc?.private_profile,
+      liked_projects: userDoc?.liked_projects,
+      banned: userDoc?.banned,
+    });
+  } catch (err) {
+    console.log(err);
```

Comment:
Better to send an error to express (`next(error)`). You can use this to generate nice errors: https://www.npmjs.com/package/http-errors

Here it's also worse than before since you don't even res anything in case of error. Meaning if something goes wrong the request will never terminate.